yaml/builders/add_dependencies_remote.yaml

```diff
@@ -9,7 +9,7 @@
 
             curl --silent https://raw.githubusercontent.com/sclorg/rhscl-container-ci/master/configuration | while read scl namespace gituser gitproject trigger hub_namespace; do
 
-              if [[ "${{trigger}}" == *{name}* ]]; then
+              if [[ "${{trigger}}" == *{name}* ]] && echo "${{ghprbCommentBody}}" | grep -qv "\[-.*${{scl}}.*\]"; then
```

Comment:
When there's already one '[[ ... ]]', I would probably go with another one instead of `| grep `. Otherwise it looks like it should work :-)

Reply:
Thanks for a quick review. I've realized that there will be problem with image building&pushing when some PR with skipped tests get merged. See https://github.com/sclorg/s2i-base-container/pull/168#issuecomment-390631282